### PR TITLE
Birth month graph

### PR DESCRIPTION
**Pull request by lukeschunk**
### Added Birth-Month Chart - plots births/month/generation
- Added a service to manipulate the raw data (currently raw data is hard coded into service for demo purposes)
- D3 is done on the directive
- View manipulations are done via the directive template and the controller

### CODE REVIEW

File: main/client/features/chart-birth-month/birth-month-chart-ctrl.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    $scope.totalBirthsPerMonth = birthMonthService.totalBirthsPerMonth;
+
+    //Load TotalBirthsPer Generation from Service
+    $scope.totalBirthsPerGeneration = birthMonthService.totalBirthsPerGeneration;
+
+    //MONTH VIEW LOGIC ====================================================
+	//MONTH VIEW LOGIC ====================================================
+	//MONTH VIEW LOGIC ====================================================
+
+    $scope.janChecked = function() {
```
Comment (r-walsh) [inline main/client/features/chart-birth-month/birth-month-chart-ctrl.js:13]: Maybe see if you can condense these into one function if you have time. Lots of repeated logic.

File: main/client/features/chart-birth-month/birth-month-chart-dir.js [hunk 2]
```diff
@@ -0,0 +1,446 @@
+angular.module('app').directive('birthMonth', birthMonthService => {
+
+    return {
+        restrict: 'EA',
```
Comment (r-walsh) [inline main/client/features/chart-birth-month/birth-month-chart-dir.js]: comma first

File: main/client/features/chart-birth-month/birth-month-chart-dir.js [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+
+            path.classed("main", true);
+
+            //MONTH SELECTORS MONTH SELECTORS MONTH SELECTORS
+            //MONTH SELECTORS MONTH SELECTORS MONTH SELECTORS
+            //MONTH SELECTORS MONTH SELECTORS MONTH SELECTORS
+            //MONTH SELECTORS MONTH SELECTORS MONTH SELECTORS
+            //MONTH SELECTORS MONTH SELECTORS MONTH SELECTORS
+
+            var janSelector = false,
```
Comment (r-walsh) [inline main/client/features/chart-birth-month/birth-month-chart-dir.js]: comma first

File: main/client/features/chart-birth-month/birth-month-chart-ctrl.js (hunk 1 above)
Comment (lukeschunk) [inline main/client/features/chart-birth-month/birth-month-chart-ctrl.js:13]: i know i suck at coding :neckbeard:

**Pull request merged**